portal-web/docroot/html/taglib/aui/nav_item/end.jsp

```diff
@@ -31,10 +31,10 @@ if (Validator.isNull(title)) {
 %>
 
 <c:if test="<%= !dropdown || Validator.isNotNull(bodyContentString.trim()) %>">
-	<li class="<%= cssClass %><%= selected ? " active " : StringPool.SPACE %><%= state %>" id="<%= id %>" role="presentation" <%= AUIUtil.buildData(data) %> <%= InlineUtil.buildDynamicAttributes(dynamicAttributes) %>>
+	<li class="nav-item <%= cssClass %><%= selected ? " active " : StringPool.SPACE %><%= state %>" id="<%= id %>" role="presentation" <%= AUIUtil.buildData(data) %> <%= InlineUtil.buildDynamicAttributes(dynamicAttributes) %>>
```

Comment:
I don't think we can add `nav-item` here by default. There are cases when rendering a dropdown menu `aui:nav-item`s are nested and `aui:nav-item` is used to render `li` elements in a dropdown-menu.

Reply:
Thank you Patrick! I'm going to take a look at it

Reply:
I've added a condition to avoid the default classes for dropdowns